src/daemon/utils_cache.h

```diff
@@ -35,6 +35,7 @@
 #define STATE_WARNING 1
 #define STATE_ERROR 2
 #define STATE_MISSING 15
+#define STATE_UNKNOWN 99
```

Comment:
These defines should be purely internal and it should be harmless to renumber them – zero would be much more appropriate for `UNKNOWN` imho ;)

Reply:
Good point,  I just updated it in the latest change. It actually helps to keep the logic while evaluating thresholds for unkown to be lower than OKAY.

I left MISSING in 15 as it is a _weird_ state, it is never reflect in a value present on the cache.